UniversalFramework_Base.xcconfig

```diff
@@ -6,16 +6,20 @@
 //
 
 // Make it universal
-SUPPORTED_PLATFORMS                    = macosx iphonesimulator iphoneos watchos watchsimulator
+SUPPORTED_PLATFORMS                    = macosx iphonesimulator iphoneos watchos watchsimulator appletvos appletvsimulator
 VALID_ARCHS[sdk=macosx*]               = i386 x86_64
 VALID_ARCHS[sdk=iphoneos*]             = arm64 armv7 armv7s
 VALID_ARCHS[sdk=iphonesimulator*]      = arm64 armv7 armv7s
 VALID_ARCHS[sdk=watchos*]              = armv7k
 VALID_ARCHS[sdk=watchsimulator*]       = armv7k
+VALID_ARCHS[sdk=appletv*]              = arm64
+VALID_ARCHS[sdk=appletvsimulator*]     = arm64
 
 // Dynamic linking uses different default copy paths
-LD_RUNPATH_SEARCH_PATHS[sdk=macosx*]          = $(inherited) '@executable_path/../Frameworks' '@loader_path/Frameworks'
-LD_RUNPATH_SEARCH_PATHS[sdk=iphoneos*]        = $(inherited) '@executable_path/Frameworks' '@loader_path/Frameworks'
-LD_RUNPATH_SEARCH_PATHS[sdk=iphonesimulator*] = $(inherited) '@executable_path/Frameworks' '@loader_path/Frameworks'
-LD_RUNPATH_SEARCH_PATHS[sdk=watchos*]         = $(inherited) '@executable_path/Frameworks'
-LD_RUNPATH_SEARCH_PATHS[sdk=watchsimulator*]  = $(inherited) '@executable_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=macosx*]           = $(inherited) '@executable_path/../Frameworks' '@loader_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=iphoneos*]         = $(inherited) '@executable_path/Frameworks' '@loader_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=iphonesimulator*]  = $(inherited) '@executable_path/Frameworks' '@loader_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=watchos*]          = $(inherited) '@executable_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=watchsimulator*]   = $(inherited) '@executable_path/Frameworks'
+LD_RUNPATH_SEARCH_PATHS[sdk=appletvos*]        = $(inherited) '@executable_path/Frameworks'
```

Comment:
Lots of this is really new to me :) Why wasn't `@loader_path` needed for watchOS?